### PR TITLE
Bug #6974 - Hitting Cancel button on New Org > Manually Assign page goes...

### PR DESCRIPTION
**Pull request by boaz0**
... to 404 page

### CODE REVIEW

**Comment by boaz0** [comment]
I think this is the fastest way to fix that problem. However, I don't mind to learn how to route a GET request. :wink:

**Comment by dLobatog** [comment]
[test]

File: app/views/hosts/_assign_hosts.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
   </div>
 </div>
   <%= content_tag(:div, :class => "form-actions") do %>
-    <%= link_to(_("Cancel"), "#{controller_name}_path", :class => "btn btn-default") %>
+    <%= link_to(_("Cancel"), :back, :class => "btn btn-default") %>
```
Comment (ohadlevy) [inline app/views/hosts/_assign_hosts.html.erb:37]: consider using the link_to_back helper instead?

Comment (boaz0) [inline app/views/hosts/_assign_hosts.html.erb:37]: @ohadlevy - yeah sure! <strike>just wondering where is it defined? i googled and greped it but haven't found any useful result.</strike> 10x.

Nevermind, found some!

Comment (boaz0) [inline app/views/hosts/_assign_hosts.html.erb:37]: OK, I get the following trace when `link_to_back(_("Cancel"), { :class=>'btn btn-default' })`:

```
NoMethodError
undefined method `link_to_back' for #<#<Class:0x000000078ac7d8>:0x0000000722e5a0>
app/views/hosts/_assign_hosts.html.erb:38:in `block (2 levels) in _app_views_hosts__assign_hosts_html_erb__3957567925500005721_42723900'
app/views/hosts/_assign_hosts.html.erb:37:in `block in _app_views_hosts__assign_hosts_html_erb__3957567925500005721_42723900'
app/helpers/layout_helper.rb:250:in `form_for'
app/views/hosts/_assign_hosts.html.erb:5:in `_app_views_hosts__assign_hosts_html_erb__3957567925500005721_42723900'
app/views/hosts/assign_hosts.html.erb:3:in `_app_views_hosts_assign_hosts_html_erb__1850536344537136393_51305740'
app/controllers/concerns/foreman/controller/taxonomies_controller.rb:139:in `assign_hosts'
app/models/concerns/foreman/thread_session.rb:33:in `clear_thread'
lib/middleware/catch_json_parse_errors.rb:9:in `call' 
```

Comment (dLobatog) [inline app/views/hosts/_assign_hosts.html.erb:37]: I don't think that helper function is available in Foreman, couldn't find it as part of Rails or any gem we use. Whoever wrote this I think meant something like `eval("#{controller_name}_path".to_sym)` and that'd work, but :back is simpler and nicer. Merging in 3..2..1.. :bomb:

**Comment by dLobatog** [comment]
Merged as 1f63475205f367a5ec01ebd8256ec4c546e6acfa , thanks @ripcurld00d !

**Pull request closed without merging**